grails-app/services/streama/TheMovieDbService.groovy

```diff
@@ -20,8 +20,16 @@ class TheMovieDbService {
 
 
   def getSimilarMovies(movieId){
-    def JsonContent = new URL(BASE_URL + "/movie/$movieId/similar?api_key=$API_KEY").text
-    return new JsonSlurper().parseText(JsonContent)
+    def JsonContent_similar = new URL(BASE_URL + "/movie/$movieId/similar?api_key=$API_KEY").text
+    def json_similar = new JsonSlurper().parseText(JsonContent_similar)
+    for(item in json_similar.results ){
```

Comment:
This isnt null-safe, json_similar might be null at this point. a way to be null-safe is to use `?`between the `.`, like so: `json_similar?.results`